radio/src/gui/common/stdlcd/draw_functions.cpp

```diff
@@ -407,6 +407,30 @@ void drawSensorCustomValue(coord_t x, coord_t y, uint8_t sensor, int32_t value,
           }
         }
       }
+      else if (telemetrySensor.id >= RB3040_OUTPUT_FIRST_ID && telemetrySensor.id <= RB3040_OUTPUT_LAST_ID) {
+        if (telemetrySensor.subId == 0) {
+          if(value == 0) {
+            lcdDrawText(x, y, "OK", flags);     
+          }
+          else {
+            for (uint8_t i=0; i<9; i++) {
```

Comment:
In 99% cases value should be 0, right? If yes I would enter into this loop only if `value != 0`